### PR TITLE
feat: Added primal.net to nostr section

### LINKED ISSUE #2206: [Feature] primal.net on nostr section
### Feature description

Add https://primal.net/ on Nostr section

### Describe the solution

Add https://primal.net/ on Nostr section

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by Tejasp2003**
### Describe the changes you have made in this PR

Now, primal.net will appear in Nostr section

### Link this PR to an issue [optional]

Fixes #2206

### Type of change

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes [optional]

image

### How has this been tested?

Manually

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/screens/Discover/websites.json [hunk 1]
```diff
@@ -153,6 +153,12 @@
         "subtitle": "The blue bird experience for Nostr",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/yosup_thumbnail.svg",
         "url": "https://yosup.app/"
+      },
+      {
+        "title": "Primal",
+        "subtitle": "Nostr highlights by Primal",
```
Comment (escapedcat) [inline src/app/screens/Discover/websites.json:159]: This should be changed or left out.  
The current subtitle is a label from their UI:
image

**Review by escapedcat — CHANGES_REQUESTED** [review]

File: src/app/screens/Discover/websites.json (hunk 1 above)
Comment (Tejasp2003) [reply src/app/screens/Discover/websites.json]: Changes updates

Comment (Tejasp2003) [reply src/app/screens/Discover/websites.json]: Okay, i am updating the subtitle

**Review by im-adithya — APPROVED** [review]

**Review by escapedcat — APPROVED** [review]

**Comment by escapedcat** [comment]
Congrats @Tejasp2003 !

**Comment by Tejasp2003** [comment]
@escapedcat Thanks :)

**Pull request merged**